truss/templates/server/common/truss_server.py

```diff
@@ -25,13 +29,42 @@ async def parse_body(request: Request) -> bytes:
     return await request.body()
 
 
-class BasetenEndpoints(V1Endpoints):
+FORMAT = "%(asctime)s.%(msecs)03d %(name)s %(levelname)s [%(funcName)s():%(lineno)s] %(message)s"
```

Comment:
I think we want to centralize all our logging formatters in `truss.templates.server.common.logging.setup_logging`.  The existing code in there should take care of this basic config already.  I think we can just remove this logging setup all together.

Reply:
If you're confident about removing this whole thing, I'm happy to do that. For reference, the default uvicorn config if we don't pass anything in looks like this:

```

LOGGING_CONFIG: Dict[str, Any] = {
    "version": 1,
    "disable_existing_loggers": False,
    "formatters": {
        "default": {
            "()": "uvicorn.logging.DefaultFormatter",
            "fmt": "%(levelprefix)s %(message)s",
            "use_colors": None,
        },
        "access": {
            "()": "uvicorn.logging.AccessFormatter",
            "fmt": '%(levelprefix)s %(client_addr)s - "%(request_line)s" %(status_code)s',  # noqa: E501
        },
    },
    "handlers": {
        "default": {
            "formatter": "default",
            "class": "logging.StreamHandler",
            "stream": "ext://sys.stderr",
        },
        "access": {
            "formatter": "access",
            "class": "logging.StreamHandler",
            "stream": "ext://sys.stdout",
        },
    },
    "loggers": {
        "uvicorn": {"handlers": ["default"], "level": "INFO", "propagate": False},
        "uvicorn.error": {"level": "INFO"},
        "uvicorn.access": {"handlers": ["access"], "level": "INFO", "propagate": False},
    },
}

```